internal/datastore/crdb/tx_test.go

```diff
@@ -0,0 +1,163 @@
+//go:build ci
+// +build ci
+
+package crdb
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/jackc/pgconn"
+	"github.com/jackc/pgx/v4"
+	"github.com/stretchr/testify/require"
+
+	"github.com/authzed/spicedb/internal/datastore/crdb/migrations"
+	"github.com/authzed/spicedb/pkg/migrate"
+	"github.com/authzed/spicedb/pkg/namespace"
+	"github.com/authzed/spicedb/pkg/secrets"
+)
+
+const (
+	testUserNamespace = "test/user"
+)
+
+var testUserNS = namespace.Namespace(testUserNamespace)
+
+// Copied from crdb_test.go
+func (st sqlTest) newCRDB() (*crdbDatastore, error) {
```

Comment:
Can we share it somehow?

Reply:
If we can, I'd like to share the util, even if it means returning two handles

Reply:
Ok I can update and use the existing one